src/frontend/context/SimuleringContext.tsx

```diff
@@ -98,17 +101,21 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
     };
     const erMigreringMedStoppISimulering = erMigreringFraInfotrygd && skalStoppeISimulering();
 
-    const erNegativeMånedsbeløpPåMaksEnKrone = (simPerioder: ISimuleringPeriode[]) =>
+    const erAvvikIMånedsbeløpPåMaksEnKronePerBarn = (
+        simPerioder: ISimuleringPeriode[],
+        antallBarn: number
+    ) =>
         simPerioder
             .filter(periode => erFør(kalenderDato(periode.fom), kalenderDato(mars2023)))
             .map(periode => periode.resultat || 0)
-            .every(beløp => beløp <= 0 && beløp >= -1);
+            .every(beløp => Math.abs(beløp) <= antallBarn);
```

Comment:
Strengt tatt åpner dette for 5kr avvik på ett barn og 0kr avvik på fire andre, men i praksis vil vel ikke det være mulig så lenge hvert beløp er satsbasert 🤔

Reply:
Strengt tatt, men jeg tror ikke det er så nøye, siden det i etterbetalingsjekken for helmanuell migrering egentlig er satt en grense på 22 kr per barn, men beløpet vi bruker er 220, så jeg tror det bare er grovt regnet med at ingen har flere enn 10 barn elns 🤷